tests/config/test_ursula_config.py

```diff
@@ -0,0 +1,41 @@
+from constant_sorrow.constants import NO_KEYRING_ATTACHED, CERTIFICATE_NOT_SAVED
+from nucypher.characters.lawful import Ursula
+from nucypher.config.characters import UrsulaConfiguration
+
+
+from nucypher.config.storages import ForgetfulNodeStorage
+
+
+# @pytest.mark.parametrize("federated_only", [True, ])  TODO
+def test_ursula_development_configuration(federated_only=True):
+    config = UrsulaConfiguration(dev_mode=True, federated_only=federated_only)
+    assert config.is_me is True
+    assert config.dev_mode is True
+    assert config.keyring == NO_KEYRING_ATTACHED
+    assert config.provider_uri == UrsulaConfiguration.DEFAULT_PROVIDER_URI
+
+    # Produce an Ursula
+    ursula_one = config()
+
+    # Ensure we do in fact have an Ursula here
+    assert isinstance(ursula_one, Ursula)
+    assert len(ursula_one.checksum_public_address) == 42
+    assert ursula_one.federated_only is federated_only
+
+    # A Temporary Ursula
+    assert ursula_one.datastore.engine.url.database == ":memory:"
+    assert ursula_one.certificate_filepath is CERTIFICATE_NOT_SAVED
```

Comment:
Just curious...shouldn't this be an equality comparison?
```suggestion
    assert ursula_one.certificate_filepath == CERTIFICATE_NOT_SAVED
```

Reply:
No - I really want to know that this value `is` the constant `CERTIFICATE_NOT_SAVED`